### DISCUSSION TITLE
Moving window and cycle pause on MBTN_LEFT issue still occurs

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#14774] [General]
Linux Debian based distro / GTK based Desktop Environment
MPV  0.38.0

I am still experiencing the issue mentioned on the following link.
Moving window and cycle pause on MBTN_LEFT 
https://github.com/mpv-player/mpv/issues/7563

I think this says at the end of post that this was fixed.
I have my LMB set to cycle pause when clicking on the video pane, which works
`MBTN_LEFT      cycle pause         # toggle pause/playback mode`

but if I hold down my LMB and drag to move the MPV window it pauses the media if anything is playing and then allows the move.

Maybe this has not made it into my version (0.38.0) yet?
Can anyone confirm if this works for you or not.

Thank You

**Comment by hooke007** [comment]
Always try git version.

**Comment by geextahslex** [comment]
I can confirm that it works on mpv v0.38.0-717 Windows 10 22H2
